### DISCUSSION TITLE
Common workflow (as opposed to shared or reusable)?

### DISCUSSION THREAD

**Discussion by dpo** [community/community#35462] [Actions]
Hi. I have an organization with ~60 repositories that all share identical workflows. I read about starter and reusable workflows. I do see the advantage of not repeating the details of the workflows, but reusable workflows still require me to add new workflows to each of the 60 repos that "call" the reusable workflows.

I wonder if there is a concept of a "common" or "organization-wide" workflow; one that would live in the `.github` repository (say) and would automatically apply to all repositories in my organization (without the need to write yet another workflow that calls it)?

Thanks.

**Comment by masonjlegere** [comment]
Not an ideal solution but we've managed to get something similar through organization  web hooks that trigger a reusable workflow sitting in a centralized place. It bridge the gap between the trigger and the workflow dispatch we have a lambda
